src/documents/tutorial-creatingAPreferencesEditorUsingThePreferencesFramework/PrefsFramework.md

```diff
@@ -0,0 +1,775 @@
+---
+title: Preferences Framework Tutorial
+layout: default
+category: Tutorials
+---
+
+This tutorial will walk you through the process of building a Preference Editor using the
+Infusion [Preferences Framework](../PreferencesFramework.md).
+
+<div class="infusion-docs-note"><strong>Note:</strong> This tutorial is not yet complete.
+Where more information is still to come, this will be clearly noted.</div>
+
+## Introduction ##
+A Preferences Editor is a web application that supports the creation and modification
+of a user's preferred settings for a device or application. In this tutorial, we'll look at a functional – but very simple – Preference Editor and explain how it
+works. From there, we’ll learn about more features of the Preferences Framework by adding functionality to the Editor.
+
+Throughout this tutorial, you’ll find links into the documentation for various parts of
+Infusion and the Preferences Framework. You shouldn’t need to visit these links to
+follow the tutorial; they’re there in case you’re interested in reading more about something.
+
+### Example code ###
+The source code used in this tutorial is one of the examples provided in the Infusion code base,
+which you can download from github: https://github.com/fluid-project/infusion. You’ll find the code
+for the simple Preference Editor in the examples folder:
+https://github.com/fluid-project/infusion/tree/master/examples/framework/preferences/minimalEditor.
+We recommend you download the Infusion library and load the example code into your favourite editor.
+<figure>
+![the folder hierarchy of the samepl code](../images/prefsEditorFolders.png)
+<figcaption>Figure 1: Folder hierarchy for the Preference Editor example</figcaption>
+</figure>
+
+The example code is a Preference Editor for the world’s first flying car.
+If you run a local webserver
+(for example using [this approach](http://www.linuxjournal.com/content/tech-tip-really-simple-http-server-python),
+or using [MAMP](https://www.mamp.info/en/))
```

Comment:
I was looking at phaser.io over the weekend and they have this page about web servers http://phaser.io/tutorials/getting-started/part2 I'm thinking it would be good to include the nodjs and/or the grunt options. Which are nice if you want to stick with javascript based servers. What do you think?